_posts/2015-08-11-tsuru-images-with-packer.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+status: publish
+published: false
+title: tsuru easier to test (spoiler: packer is awesome).
+author:
+  display_name: Tarsis Azevedo
+  url: https://github.com/tarsisazevedo
+date: '2015-08-11 14:30:00 -0300'
+date_gmt: '2015-08-11 14:30:00'
+categories:
+- Images
+tags:
+- images packer
+comments: []
+---
+
+One of our goals is make tsuru easy to people who wants to try it for
+the first time. To do it we are creating all-in-one aws and virtualbox vagrant
+box images of tsuru with [packer](https://packer.io). These images are based on
+stable and nightly packages.
```

Comment:
s/make/making/ in the first sentence.

Also, you can start the second sentence with "In order to achieve this goal, we are creating..." instead of "To do it".

s/creating/offering/ (in order to offer, we need to create it, but "offering" sounds better)
s/aws/AWS/
s/virtualbox/VirtualBox/
s/packages/releases/